packages/core/src/utils.ts

```diff
@@ -161,15 +161,14 @@ npx hardhat chugsplash-deploy --network ${networkName} ${configPath}
   }
 
   if (requiresOwnershipTransfer.length > 0) {
-    // TODO update this once the transfer ownership task is implemented
     throw new Error(
       `Error: Detected proxy contracts which are not managed by ChugSplash.
       ${requiresOwnershipTransfer.map(
         ({ name, address }) => `${name}, ${address}\n`
       )}
 
 To upgrade these contracts, you must first transfer ownership of them to ChugSplash using the following command:
-npx hardhat chugsplash-transfer-ownership>
+npx hardhat chugsplash-transfer-ownership --network ${networkName} ${configPath} --proxy <proxy address>
```

Comment:
This'll need to be updated to `--config ${configPath}` since it's now a regular param